### DISCUSSION TITLE
Bestehende CCU3 in HomeAssistant einbinden

### DISCUSSION THREAD

**Discussion by KosoTheOne** [OpenCCU/OpenCCU#2041] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,

ich habe eine bestehende CCU3 daheim und wollte diese nun in meinen HomeAssistant einbinden. ich stoße dabei immer wieder auf das RaspberryMatic Addon. Nur finde ich keinen weg wie das gehen Soll. Ist das überhaupt möglich?

Danke für eure Hilfe

**Comment by mallebuchsbaum** [comment]
Kommt drauf an was du willst. Nur die CCU in HA einbinden? Oder die CCU durch Rasperrymatic ersetzen. 
Zu beidem gibt massig Videos auf YT. Wenn es ums ersetzen geht: in der bestehenden CCU ein Backup machen und dann in der Rasperrymatic wieder einspielen.

**Comment by KosoTheOne** [comment]
ich habe es hinbekommen. Mit HomemtaicIP Local

**Comment by jens-maus** [comment, marked as answer]
Genau so ist es. Man muss im Grunde (auch wenn man eine reine CCU3 und kein RaspberryMatic einsetzt) wie hier beschrieben vorgehen:

https://github.com/jens-maus/RaspberryMatic/wiki/HomeAssistant-Integration

**Comment by steven83ger** [comment]
Also man kann eine CCU3 ohne RaspberryMatic einbinden?
Bei der Einrichtung von HomematicIP local stoße ist auf das Problem dass ich in HomematicIP keinen Benuternamen und Passwort habe.
Hatte bisher Homematic IP nur mit dem Smartphone konfituriert. habe jetzt gefunden dass auch bei Homematic IP ein WebIU über den Browser möglich ist. Nur leider passiert bei mir garnichts wenn ich die IP Adresse der CCU3 im Browser eingebe.
VG

**Reply by jens-maus** [reply]
Also für mich liest sich das hier das du hier homematicIP via Cloud mit homematicIP via CCU3 durcheinander bringst und vielleicht sogar denkst dein `HmIP-HAP` der für die Verbindung zur Cloudlösung von homematicIP da ist wäre eine CCU3 (eben weil die fast gleich aussehen). Also bitte erst einmal überprüfen was du da konkret einsetzt und mal auf die Hinterseite der "CCU3" mal schauen ob bei dem Typenschild auch wirklich "CCU3" draufsteht. Denn die "Homematic(IP) Local" Integration in HomeAssistant benötigt definitiv eine CCU3 bzw. RaspberryMatic Lösung!

**Reply by steven83ger** [reply]
Hallo Jens, vielen Dank für die schnelle Antwort. 
Ja so ist es, ich habe nur den Access-Point für die Cloud. Jetzt werde ich erst einmal überlegen wie ich für mich am besten vorgehe.
VG

**Comment by Burak3850** [comment]
Hallo Zusammen,
ich habe folgendes Problem, bei der Anmeldung von HA an ccu3. Die Anmeldung erfolgt ohne Benutzername und Kennwort bei der CCU3. Egal was ich da eintrage, verbindet sich Home Assistent mit Home Matic nicht. 
Ich kann über Web UCI ganz normal auf meine CCU3 zugreifen. Wisst ihr vielleicht, woran das liegt, dass ich Home Assistent mit Home Matic nicht verbinden kann?

Gruß Burak

**Reply by Baxxy13** [reply]
Leg dir über die WebUI einen neuen Benutzer (mit Adminrechten) samt Passwort (ohne Sonderzeichen/Umlaute) an und nutze dann diesen User für die Integration.

**Reply by Friedel65** [reply]
Funktioniert bei mir auch nicht. Ich hoffe das ich irgendwo einen Fehler gemacht habe den ich finden kann

**Reply by Friedel65** [reply]
muß in der ccu3 nicht auch ein addon instaliiert werden wie für cux demmon oder sonstige sondersoftware

**Reply by Baxxy13** [reply]
Nein, es werden keine AddOns oder sonstige "Extras" auf der CCU/RM benötigt.
